quic/s2n-quic-core/Cargo.toml

```
@@ -33,6 +33,7 @@ tracing = { version = "0.1", default-features = false, optional = true }
 zerocopy = "=0.6.0"
 zerocopy-derive = "=0.3.0"
 futures-test = { version = "0.3", optional = true } # For testing Waker interactions
+once_cell = { version = "1", default-features = false }
```

Looks like `once_cell::sync` requires std, so you'll need to make this an optional dependency, and add it to the `std` feature on line 15:
```suggestion
once_cell = { version = "1", optional = true }
```

```
std = ["alloc", "once_cell"]
```